pkg/controller/direct/compute/forwardingrule/forwardingrule_controller.go

```diff
@@ -106,8 +106,7 @@ func (m *forwardingRuleModel) AdapterForObject(ctx context.Context, reader clien
 	// Get location
 	location := obj.Spec.Location
 
-	// Set label managed-by-cnrm: true
-	obj.ObjectMeta.Labels["managed-by-cnrm"] = "true"
+	
```

Comment:
Would be nice if it left 1 blank line here instead of 3. Otherwise very nice :)

Reply:
Agreed, will fix.  Pretty sure this is going to fail tests though (so I'd like to let it run), and I'm trying to figure out the "why" here.